src/utils/layout-helpers.ts

```diff
@@ -1,11 +1,14 @@
 import * as memoize from 'memoizee';
 import * as path from 'path';
+import * as childProcess from 'child_process';
```

Comment:
We can't rely on `child_process`, because server may be running inside worker.

Reply:
reverted